includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -1096,6 +1145,24 @@ public static function print_status_meta_box( $post ) {
 					</div>
 
 					<div class="misc-pub-section">
+						<?php
+						$staleness = self::get_post_staleness( $post );
+						if ( ! empty( $staleness ) ) {
+							echo '<div class="notice notice-info notice-alt inline"><p>';
+							echo '<b>';
+							esc_html_e( 'Stale results', 'amp' );
+							echo '</b>';
+							echo '<br>';
+							if ( ! empty( $staleness['theme'] ) && ! empty( $staleness['plugins'] ) ) {
+								esc_html_e( 'Different theme and plugins were active when these results were obtained. Please re-check.', 'amp' );
+							} elseif ( ! empty( $staleness['theme'] ) ) {
```

Comment:
@westonruter The `'theme'` key may not exist in the returned `$staleness` array.  We either need to initialize `$staleness` with this key back here on [line 479)[https://github.com/Automattic/amp-wp/pull/1375/files#diff-57518db0811ee918fee17d27459ed4ecR479] or check `isset()`.

Reply:
Actually, `empty()` and `isset()` are both PHP language constructs which will not complain about undefined array indexes. So using `isset( $staleness['theme'] ) && (bool) $staleness['theme']` is functionally equivalent to `! empty( $staleness['theme'] )`.

The absence of the `theme` key is important for the return value to satisfy an empty condition, as `array( 'theme' => array() )` is not `empty` for the overall check above in:

https://github.com/Automattic/amp-wp/blob/54b1272d05060d70d3ab2ace696fd9f7b3e540ed/includes/validation/class-amp-invalid-url-post-type.php#L1149-L1150

Reply:
@westonruter True. Valid point. Thank you.